tests/kubernetes/test_pod_generator.py

```diff
@@ -473,6 +480,9 @@ def test_construct_pod_empty_executor_config(self, mock_uuid):
         worker_config.metadata.labels['app'] = 'myapp'
         worker_config.metadata.name = 'pod_id-' + self.static_uuid.hex
         worker_config.metadata.namespace = 'namespace'
+        worker_config.spec.containers[0].env.append(
+            k8s.V1EnvVar(name="AIRFLOW_IS_K8S_EXECUTOR_POD", value='True')
+        )
```

Comment:
This is static -- it should be added in to `pod_generator_base_with_secrets.yaml` instead of being added via code.
```suggestion
```

Reply:
@ashb This value is different as it is applied by the `construct_pod` method. I don't think it belongs in the pod_template_file as that would be confusing to anyone reading the code.

Reply:
Oh yes, you're right. I thought the file was the expected output, not the input. :+1: